EventFilter/HcalRawToDigi/src/HcalUnpacker.cc

```diff
@@ -699,9 +712,9 @@ void HcalUnpacker::unpackUTCA(const FEDRawData& raw, const HcalElectronicsMap& e
 	int ichan=(i.channelid()&0x3);
 	HcalElectronicsId eid(crate,slot,ifiber,ichan, false);
 	DetId did=emap.lookup(eid);
-	
+
 	if (!did.null()) { // unpack and store...
-	  if (did.det()==DetId::Calo && did.subdetId()==HcalZDCDetId::SubdetectorId) {
+	  if (HcalRawToDigi::unpackZDCQIE10_ == true && did.det()==DetId::Calo && did.subdetId()==HcalZDCDetId::SubdetectorId) {
```

Comment:
no need to say ``` == true ``` with booleans, although it's equivalent.
I was hoping still for a configuration parameter so that you could do your tests without recompilation if the default value is false.
Recall that this flag setting story started with the purpose of having it set to false in 80X until we see good data from RPD.